core/vtk/ttkExtract/ttkExtract.cpp

```diff
@@ -496,7 +496,12 @@ int ttkExtract::ExtractGeometry(vtkDataObject *output,
     threshold->SetInputDataObject(maskOutput);
     threshold->SetInputArrayToProcess(
       0, 0, 0, isPointDataArray ? 0 : 1, "Mask");
+#if VTK_MAJOR_VERSION > 9 || (VTK_MAJOR_VERSION >= 9 && VTK_MINOR_VERSION >= 2)
```

Comment:
There is `VTK_VERSION_CHECK` to do this. The replacement has been there since 9.1

Reply:
indeed, that is cleaner. Fixed, thanks !

Reply:
More specifically:

```cpp
#if VTK_VERSION_NUMBER < VTK_VERSION_CHECK(9, 1, 0)
  // old code
#else
  // new code
#endif
```

Reply:
Unfortunately, it seems the **VTK** provided by ParaView 5.8.1 does not yet support this feature. The CI complains:
```
/home/vsts/work/1/s/core/vtk/ttkExtract/ttkExtract.cpp:498:44: error: missing binary operator before token "("
 #if VTK_VERSION_NUMBER >= VTK_VERSION_CHECK(9, 2, 0)
```

Reply:
Your version indeed work with PV 5.8.1, this MR should be ready once the pipeline is green :)